node/chainSimulator/components/testOnlyProcessingNode.go

```diff
@@ -151,11 +151,6 @@ func NewTestOnlyProcessingNode(args ArgsTestOnlyProcessingNode) (*testOnlyProces
 	if err != nil {
 		return nil, err
 	}
-	err = instance.createTransactionLogProcessor()
```

Comment:
We need the tx logs processor. Please put back the code

Reply:
the logs processor is created inside the process components.

Reply:
ok